modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -3729,30 +3731,34 @@ public void testUpdateObjectEntry() throws Exception {
 
 		_assertLocalizedValues(
 			HashMapBuilder.<String, Object>put(
-				"localizedLongTextObjectFieldName", ""
+				"localizedLongTextObjectFieldName",
+				"en_US localizedLongTextObjectFieldValue"
 			).put(
 				"localizedRichTextObjectFieldName",
 				"ar_SA localizedRichTextObjectFieldValue"
 			).put(
 				"localizedRichTextObjectFieldNameRawText",
 				"ar_SA localizedRichTextObjectFieldValue"
 			).put(
-				"localizedTextObjectFieldName", ""
+				"localizedTextObjectFieldName",
+				"en_US localizedTextObjectFieldValue1"
```

Comment:
Hey @pedro-oliveira446 , could you please help me to make sure that we have other examples in the portal where makes sense to retrieve data like this, I mean a part of the data in one language and other part in other language.

If we have examples of it this pull request looks good and can be forward.

cc: @linolaoj

Reply:
I discussed this behavior with Luis, and we based our approach on the Web Content component. Please see the video above for reference.


https://github.com/user-attachments/assets/4e196457-cbf4-43bb-9a66-2224239d8390